Panic using RandomString when specifying symbols

I have a table with a varchar(255) column for which I want to generate a random ID while dumping (this column only has NULL values in the original database). Here's the config I try to use

```yaml
    - schema: public
      name: surgery_patient
      transformers:
        - name: RandomString
          params:
            column: permanent_identification_number
            symbols: 0123456789
            min_length: 20
            max_length: 20
            keep_null: false
```
When running dump or validate, greenmask fails with the following error
```
greenmask --config greenmask.yml validate   --warnings   --data   --diff   --schema   --format=text   --table-format=vertical   --transformed-only   --rows-limit=1

panic: runtime error: index out of range [9] with length 9

goroutine 185 [running]:
github.com/greenmaskio/greenmask/internal/db/postgres/transformers/utils.RandomString(0xc000538b48?, 0x42523c?, 0x14, {0xc000708450, 0x9, 0x7f2cbbfc2ca8?}, {0xc000038140, 0x14, 0x14})
	/home/runner/work/greenmask/greenmask/internal/db/postgres/transformers/utils/transformation_funcs.go:142 +0x114
github.com/greenmaskio/greenmask/internal/db/postgres/transformers.(*RandomStringTransformer).Transform(0xc000144e70, {0x1e0?, 0xf44720?}, 0xc0004c9b60)
	/home/runner/work/greenmask/greenmask/internal/db/postgres/transformers/random_string.go:148 +0xf3
github.com/greenmaskio/greenmask/internal/db/postgres/dumpers.(*TransformationPipeline).TransformSync(0xc000814c00, {0x13afef8, 0xc0007240a0}, 0x100ffffffff?)
	/home/runner/work/greenmask/greenmask/internal/db/postgres/dumpers/transformation_pipeline.go:127 +0xa2
github.com/greenmaskio/greenmask/internal/db/postgres/dumpers.(*TransformationPipeline).Dump(0xc000814c00, {0x13afef8, 0xc0007240a0}, {0xc00073a035?, 0xc000538dc0?, 0x13aec90?})
	/home/runner/work/greenmask/greenmask/internal/db/postgres/dumpers/transformation_pipeline.go:153 +0x119
github.com/greenmaskio/greenmask/internal/db/postgres/dumpers.(*ValidationPipeline).Dump(0xc00007a068, {0x13afef8, 0xc0007240a0}, {0xc00073a035, 0xcb, 0xcb})
	/home/runner/work/greenmask/greenmask/internal/db/postgres/dumpers/validation_pipeline.go:33 +0x1c6
github.com/greenmaskio/greenmask/internal/db/postgres/dumpers.(*TableDumper).process(0xc0001244c8, {0x13afef8, 0xc0007240a0}, {0x13b52a8?, 0xc000010a98?}, {0x7f2cbbdf98a0?, 0xc0006960f0?}, {0x13b0278, 0xc00007a068})
	/home/runner/work/greenmask/greenmask/internal/db/postgres/dumpers/table.go:151 +0x3ad
github.com/greenmaskio/greenmask/internal/db/postgres/dumpers.(*TableDumper).Execute.func2()
	/home/runner/work/greenmask/greenmask/internal/db/postgres/dumpers/table.go:91 +0x305
golang.org/x/sync/errgroup.(*Group).Go.func1()
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:78 +0x56
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 152
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:75 +0x96
```

If I remove the symbols param, it works as expected

I'm using greenmask 0.1.9 on Fedora 39 (the linux-amd64 build)

--- a/internal/db/postgres/transformers/utils/transformation_funcs.go
+++ b/internal/db/postgres/transformers/utils/transformation_funcs.go
@@ -139,7 +139,7 @@ func RandomString(randGen *rand.Rand, minLength, maxLength int64, symbols []rune
 	}
 
 	for i := int64(0); i < length; i++ {
-		buf[i] = symbols[rand.Int63n(maxLength)]
+		buf[i] = symbols[rand.Int63n(maxLength)%int64(len(symbols))]
 	}
 	return string(buf[:length])
 }

--- a/internal/db/postgres/transformers/utils/transformation_funcs_test.go
+++ b/internal/db/postgres/transformers/utils/transformation_funcs_test.go
@@ -15,6 +15,7 @@
 package utils
 
 import (
+	"math/rand"
 	"testing"
 	"time"
 
@@ -79,6 +80,46 @@ func TestRandomDateTruncateDate(t *testing.T) {
 			res, err := TruncateDate(&tt.part, &tt.original)
 			require.NoError(t, err)
 			assert.Equal(t, tt.expected, *res)
+		})
+	}
+}
+
+func TestRandomString(t *testing.T) {
+	r := rand.New(rand.NewSource(time.Now().UnixMicro()))
+	tests := []struct {
+		name    string
+		min     int64
+		max     int64
+		symbols []rune
+	}{
+		{
+			name:    "min length",
+			min:     10,
+			max:     20,
+			symbols: []rune("0123456789"),
+		},
+		{
+			name:    "max length",
+			min:     10,
+			max:     20,
+			symbols: []rune("abcdefg12345678990,al-"),
+		},
+		{
+			name:    "empty symbols",
+			min:     20,
+			max:     20,
+			symbols: []rune("0123456789"),
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			buf := make([]rune, tt.max)
+			res := RandomString(r, tt.min, tt.max, tt.symbols, buf)
+			require.True(t, len(res) >= int(tt.min))
+			for idx, ch := range res {
+				assert.Containsf(t, tt.symbols, ch, "unexpected symbol [%d]=%c", idx, ch)
+			}
 		})
 	}
 }

--- a/internal/db/postgres/transformers/utils/transformation_funcs_test.go
+++ b/internal/db/postgres/transformers/utils/transformation_funcs_test.go
@@ -93,19 +93,19 @@ func TestRandomString(t *testing.T) {
 		symbols []rune
 	}{
 		{
-			name:    "min length",
+			name:    "result is shorter than sequence",
 			min:     10,
 			max:     20,
 			symbols: []rune("0123456789"),
 		},
 		{
-			name:    "max length",
+			name:    "long sequence and variadic length",
 			min:     10,
 			max:     20,
-			symbols: []rune("abcdefg12345678990,al-"),
+			symbols: []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"),
 		},
 		{
-			name:    "empty symbols",
+			name:    "static length and and short sequence",
 			min:     20,
 			max:     20,
 			symbols: []rune("0123456789"),

